src/Microsoft.OData.Client/Serialization/HttpWebRequestMessage.cs

```diff
@@ -63,149 +63,143 @@ public class HttpWebRequestMessage : DataServiceClientRequestMessage
         private bool sendingRequest2Fired;
 #endif
 
-        #endregion // Private Fields
-
-#if PORTABLELIB
         /// <summary>
+        /// Constructor for HttpWebRequestMessage.
         /// Initializes the <see cref="HttpWebRequestMessage"/> class.
         /// </summary>
-        static HttpWebRequestMessage()
+        public HttpWebRequestMessage(DataServiceClientRequestMessageArgs args) 
+            : base(args.ActualMethod)
         {
-            IsRunningOnSilverlight = typeof(System.Windows.Input.ICommand).GetAssembly().GetType("System.Net.Browser.WebRequestCreator", false) != null;
+            _requestMessage = new HttpRequestMessage();

```

Comment:
@ElizabethOkerio How come `_requestMessage` is initialized here and again on [line 82](https://github.com/OData/odata.net/pull/1857/files#diff-e096d30bc67d28536bce40ad13ad42fc0aca41d17855b147901675a27c445b96R82)?